cypress/cypress-nginx.conf

```diff
@@ -0,0 +1,131 @@
+# NGINX Conf file used by Cypress-Nginx Github actions
+# so we can run cypress tests against the local nginx server
+# with a ssl cert set on the domain
+
+#user  nobody;
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+
+    keepalive_timeout  65;
+
+    # fixes issues for large response headers
+    proxy_buffer_size   128k;
+    proxy_buffers   4 256k;
+    proxy_busy_buffers_size   256k;
+
+
+	# manage.thegulocal.com
+	# ======================
+	server {
+		listen 443 ssl;
+		server_name manage.thegulocal.com;
+		proxy_http_version 1.1; # this is essential for chunked responses to work
+
+		ssl_certificate manage.thegulocal.com.crt;
+		ssl_certificate_key manage.thegulocal.com.key;
+		ssl_session_timeout 5m;
+		ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+		ssl_ciphers HIGH:!aNULL:!MD5;
+		ssl_prefer_server_ciphers on;
+
+		location / {
+			proxy_pass http://localhost:9234/;
+			proxy_set_header Host $http_host;
+			proxy_http_version 1.1;
+			proxy_set_header Upgrade $http_upgrade;
+			proxy_set_header Connection "Upgrade";
+		}
+	}
+
+	# members-data-api.thegulocal.com
+	# ======================
+	server {
+		listen 443 ssl;
+		server_name members-data-api.thegulocal.com;
+		proxy_http_version 1.1; # this is essential for chunked responses to work
+
+		ssl_certificate members-data-api.thegulocal.com.crt;
+		ssl_certificate_key members-data-api.thegulocal.com.key;
+		ssl_session_timeout 5m;
+		ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+		ssl_ciphers HIGH:!aNULL:!MD5;
+		ssl_prefer_server_ciphers on;
+
+		location / {
+			proxy_pass https://members-data-api.code.dev-theguardian.com;
+			proxy_next_upstream error timeout http_404 non_idempotent;
+			proxy_set_header "X-Forwarded-Proto" "https";
+			proxy_set_header 		Host members-data-api.code.dev-theguardian.com;
+			proxy_set_header        Accept-Encoding "";
+			proxy_hide_header 		Content-Security-Policy;
+
+			proxy_cookie_domain 	members-data-api.code.dev-theguardian.com members-data-api.thegulocal.com;
+			proxy_cookie_domain 	.code.dev-theguardian.com .thegulocal.com;
+
+			sub_filter_types        application/json;
+			sub_filter_once         off;
+			sub_filter              'members-data-api.code.dev-theguardian.com'  'members-data-api.thegulocal.com';
+		}
+	}
+
+
+    # profile.thegulocal.com
+	# ======================
+    server {
+      listen 443 ssl;
+      server_name profile.thegulocal.com;
+      proxy_http_version 1.1; # this is essential for chunked responses to work
+
+      ssl_certificate profile.thegulocal.com.crt;
+      ssl_certificate_key profile.thegulocal.com.key;
+      ssl_session_timeout 5m;
+      ssl_protocols TLSv1.2 TLSv1.3;
```

Comment:
Why is this different to L59?

Reply:
Good catch! @coldlink we should remove `TLSv1` and `TLSv1.1` from all our local Nginx configs, it'll help debug mystery TLS errors.